src/main/groovy/com/matyrobbrt/gml/mappings/SrgParser.groovy

```diff
@@ -6,17 +6,21 @@
 package com.matyrobbrt.gml.mappings
 
 import groovy.transform.CompileStatic
+import org.codehaus.groovy.util.ListHashMap
 
 @CompileStatic
 class SrgParser implements Closeable {
 
     final Reader reader
 
     // obf class name to map of obf -> srg names
-    final Map<String, Map<String, String>> methods = new HashMap<>()
-    final Map<String, Map<String, String>> fields = new HashMap<>()
-    Map<String, String> workingMethods
-    Map<String, String> workingFields
+    // initial capacity is 9000 because on 1.19.2, max size ends up being 6859. Adding 25% to account for the Map's
+    // default load factor and rounding *up* to the nearest 250. Re-adjust this every MC release for better performance.
+    final Map<String, Map<String, String>> methods = new LinkedHashMap<>(8750)
```

Comment:
Use the second constructor which allows specifying a load factor of `1`. Same for all others, to reduce the ambiguity.

Reply:
As discussed earlier, this may have a negative impact on hashing performance and wouldn't work as nicely when running on a newer version of MC than GML was built on